tsung/templates/tsung-form-submission.xml.j2

```diff
@@ -0,0 +1,87 @@
+<?xml version="1.0"?>
+<!DOCTYPE tsung SYSTEM "{{ dtd_path }}" [
+    <!ENTITY validation SYSTEM "{{ transactions_dir }}/validation.xml">
+    <!ENTITY save_form SYSTEM "{{ transactions_dir }}/save_form.xml">
+    <!ENTITY authentication SYSTEM "{{ transactions_dir }}/authentication.xml">
+    <!ENTITY new_case SYSTEM "{{ transactions_dir }}/new_case.xml">
+    <!ENTITY update_cases SYSTEM "{{ transactions_dir }}/update_cases.xml">
+    <!ENTITY update_sync_record SYSTEM "{{ transactions_dir }}/update_sync_record.xml">
+    <!ENTITY update_sync_record_new_case SYSTEM "{{ transactions_dir }}/update_sync_record_new_case.xml">
+]>
+<tsung loglevel="info">
+    <clients>
+        <client host="localhost" use_controller_vm="true"/>
+    </clients>
+
+    <servers>
+        <server host="{{ pg_host }}" port="{{ pg_port }}" type="tcp"></server>
+    </servers>
+
+    <load>
+        <arrivalphase phase="1" duration="{{ duration }}" unit="second">
+            <users interarrival="5" unit="second"></users>
+        </arrivalphase>
+    </load>
+
+    <options>
+        <option name="file_server" id="casedb" value="{{ casedb }}"></option>
+        <option name="file_server" id="formdb" value="{{ formdb }}"></option>
+    </options>
+
+    <sessions>
+        <!-- Form submission broken down in primitives based on: https://docs.google.com/document/d/1MzQSjIyWhICRxnyEdGuW9eigD0N0Ka6p7DWcLP81204/edit#heading=h.ckpofgcu6eec -->
+        <session probability="50" name="form-submission-new-case" type="ts_pgsql">
+            <!-- Set the form -->
+            <setdynvars sourcetype="file" fileid="formdb" delimiter="," order="random">
+                <var name="user_id" />
+                <var name="domain" />
+                <var name="form_json" />
+            </setdynvars>
+
+            <!-- Each transaction gets its own statistics and reports -->
+            <transaction name="connection">
+                <request>
+                    <pgsql type="connect" database="{{ pg_database }}" username="{{ pg_username }}" />
+                </request>
+            </transaction>
+
+
+            &save_form;
```

Comment:
there's no validation here because there's no case to check if the domain/user exists